lmfdb/modular_curves/main.py

```diff
@@ -962,11 +962,12 @@ def __init__(self):
             [CPlabel],
         ]
 
+    _default = ["level", "index", "genus", "coarse_class_num", "coarse_level", "coarse_num", "fine_num"]
     sorts = [
-        ("", "level", ["level", "index", "genus", "label"]),
-        ("index", "index", ["index", "level", "genus", "label"]),
-        ("genus", "genus", ["genus", "level", "index", "label"]),
-        ("rank", "rank", ["rank", "genus", "level", "index", "label"]),
+        ("level", "level", _default),
```

Comment:
I think we want `"", "level", _default` for the first entry

Reply:
Done (although I don't think it actually matters).

Reply:
The way it matters is just in the url: if you have a blank default sort option it doesn't clutter urls.